Trouble importing version 2.1.0

Hello,

Firstly, I want to express my appreciation for the browserslist-to-esbuild package. It's been very useful in my projects.

Recently, I upgraded to version 2.1.0 and encountered an issue with the import statement in my vite.config.ts file. Here's how I'm using it:

```typescript
import browserslistToEsbuild from 'browserslist-to-esbuild';

export default defineConfig({
   build: {
    target: browserslistToEsbuild()
  },
});
```

After the upgrade, the import fails with the error: Cannot find module 'browserslist-to-esbuild' or its corresponding type declarations.

I'm not an expert with NPM packages, but I found that modifying your `package.json` as shown below resolves the issue:

```diff
{
  "name": "browserslist-to-esbuild",
  ...
  "type": "module",
- "exports": {
-   "types": "./src/index.d.ts",
-   "default": "./src/index.js"
- }
+ "exports": {
+   ".": "./src/index.js"
+ },
+ "types": "./src/index.d.ts",
   ...
}
```

I'm not sure if I should be importing the module differently or if the exports field in `package.json` needs to be specified differently. Could you please provide some guidance on this? I can help out with a PR with this changes if that helps.

Thank you for your time and assistance.

--- a/package.json
+++ b/package.json
@@ -20,6 +20,7 @@
 		"types": "./src/index.d.ts",
 		"default": "./src/index.js"
 	},
+	"types": "./src/index.d.ts",
 	"bin": {
 		"browserslist-to-esbuild": "./cli/index.js"
 	},

